CrisisTextLine/Elmer
# Moved sampleData file into a separate folder and changed all referenc…

#### What's this PR do?
Moves the sampleData file (used for testing) into its own "sample_data" folder and changes all references to sampleData to reflect that move.
#### Where should the reviewer start?
sample_data/sampleData.js
#### How should this be manually tested?
Check all references to sampleData to make sure they reference the new location.
#### Any background context you want to provide?
#### What are the relevant tickets?
[INT-166](https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-166)
#### Questions:

…es to sampleData file to match.
